src/dsp_tools/utils/excel_to_json/utils.py

```diff
@@ -0,0 +1,289 @@
+# pylint: disable=wrong-import-order
+
+from __future__ import annotations
+
+from typing import Any
+import numpy as np
+import pandas as pd
+import regex
+from unittest import mock
+
+from dsp_tools.models.exceptions import UserError
+
+languages = ["en", "de", "fr", "it", "rm"]
+
+
+def read_excel_file(excel_filename: str) -> pd.DataFrame:
+    """
+    This function reads an Excel file, if there is a ValueError then it patches the openpyxl part that creates the
+    error and opens it with that patch. It cleans and then returns the pd.DataFrame.
+
+    Args:
+        excel_filename: The name of the Excel file
+
+    Returns:
+        A pd.DataFrame
+    """
+    # load file
+    try:
+        read_df: pd.DataFrame = pd.read_excel(excel_filename)
+    except ValueError:
+        # Pandas relies on openpyxl to parse XLSX files.
+        # A strange behaviour of openpyxl prevents pandas from opening files with some formatting properties
+        # (unclear which formatting properties exactly).
+        # Apparently, the excel2json test files have one of the unsupported formatting properties.
+        # Credits: https://stackoverflow.com/a/70537454/14414188
+        with mock.patch("openpyxl.styles.fonts.Font.family.max", new=100):
+            read_df = pd.read_excel(excel_filename)
+    read_df = clean_data_frame(unclean_df=read_df)
+    return read_df
+
+
+def clean_data_frame(unclean_df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Will this function eventually replace `prepare_dataframe()`? Now we have 2 functions that do similar things. 

Related question: `prepare_dataframe()` is only used in `excel_to_json/resources.py`. I realized that `excel_to_json/resources.py` is now outdated, and uses old patterns, that are improved in `properties.py`. Perhaps, you could make a follow-up PR that assimilates the `excel_to_json/resources.py` to `excel_to_json/properties.py`? I mean only the most important stuff, only the overall-structure. You don't have to go too far into the details, because there are other things that are more important.

Reply:
I have planned to look at all of the excel_to_json files, as it makes no sense to have a new util and not use them consistently. How much we want to go into detail we can discuss at a later point if that is ok with you.